cachito/web/static/api_v1.yaml

```diff
@@ -1162,6 +1249,18 @@ components:
                   properties:
                     purl:
                       $ref: "#/components/schemas/Purl"
+    RequestState:
```

Comment:
Should `RequestUpdate.state` also be a RequestState rather than a string? (Would have the same problem with being nullable though)

Reply:
Yes, I think that makes sense. Good catch! (also see my comment about null in the PackageManager thread above)

Reply:
Hmm, yeah that's a good point. I recall thinking it made sense in some context, let me see if I can remember what context that was :smile: